src/window.cpp

```diff
@@ -2689,15 +2689,15 @@ void HandleKeypress(uint keycode, WChar key)
 /**
  * Handle the modifier key state changes.
  */
-void HandleModifierKeys(bool shift_pressed, bool ctrl_pressed) {
+void HandleModifierKeys(bool shift_pressed, bool ctrl_pressed, bool alt_pressed) {
 	bool old_fn_pressed = _fn_pressed;
 	bool old_remove_pressed = _remove_pressed;
 
 	_shift_pressed = shift_pressed;
 
-	/* Hardwire modifiers leaving remove inaccessible for now. */
+	/* Hardwire modifier keys. */
 	_fn_pressed = ctrl_pressed;
-	_remove_pressed = false;
+	_remove_pressed = alt_pressed;
```

Comment:
One massive problem, for me at least, is that Alt + dragging moves the whole window of OpenTTD for me. This is on Linux with Xorg and LXDE, but I'm seeing queries on how to disable that for other window managers.

Essentially this makes it impossible for me to drag-remove anything.

Reply:
On whatever linuxes i used alt-dragging haven't been moving windows for at least 10 years and even back then there was an option to disable. And even if you for some reason don't want to disable it and don't want to reassign useless estimate on alt you can remove with bulldozer or bulldozer hotkey, that's still the better way to remove anything without this pr as ctrl only removes what in vanilla, just rails?